stable/artifactory-ha/values.yaml

```diff
@@ -1149,10 +1149,21 @@ nginx:
   #    memory: "128Mi"
   #    cpu: "50m"
 
+  # Logs options
+  logs: {}
+    # Send error logs to stderr insead of a file
+    # stderr: true
```

Comment:
Please uncomment this part. I think its better to use:
```
logs:
  stderr: false
  level: warn
```

Reply:
I agree